lib/containers/container_images.pm

```diff
@@ -67,7 +67,7 @@ The main container runtimes do not need C<buildah> variable in general, unless
 you want to build the image with buildah but run it with $<runtime>
 
 =cut
-sub test_containered_app {
+sub test_built_image {
```

Comment:
Now there are `test_built_image` and `test_built_img`...

Reply:
I hope you agree that the previous `test_containered_app()` was also not good name.
Right now we have `test_built_image()` and then `prepare_img()`, `build_img()` and `test_img()`.

Is that OK?

Reply:
```suggestion
sub built_and_run_image {
```